src/SamplesApp/UITests.Shared/Windows_UI_Xaml/DragAndDrop/DragDrop_TestPage.xaml.cs

```diff
@@ -0,0 +1,393 @@
+
+				this.DropDetailsTextBlock.Visibility = Visibility.Collapsed;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Html))
+			{
+				title += " (Html)";
+
+				var html = await args.DataView.GetHtmlFormatAsync();
+				details = html;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Rtf))
+			{
+				title += " (Rtf)";
+
+				var rtf = await args.DataView.GetRtfAsync();
+				details = rtf;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.StorageItems))
+			{
+				title += " (StorageItems)";
+
+				var items = await args.DataView.GetStorageItemsAsync();
+				if (items.Count > 0)
+				{
+					// Only use the first item
+					foreach (IStorageItem item in items)
+					{
+						var file = item as StorageFile;
+
+						if (file != null)
+						{
+							details += "Name: " + file.Name + Environment.NewLine;
+							details += "DisplayName: " + file.DisplayName + Environment.NewLine;
+							//details += "DisplayType: " + file.DisplayType + Environment.NewLine;
+							details += "ContentType: " + file.ContentType + Environment.NewLine;
+							details += "Path: " + file.Path;
+						}
+					}
+				}
+			}
+			else if (args.DataView.Contains(StandardDataFormats.UserActivityJsonArray))
+			{
+				title += " (UserActivityJsonArray)";
+
+				// TODO
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Uri))
+			{
+				title += " (Uri)";
+
+				var uri = await args.DataView.GetUriAsync();
+				details = uri.ToString();
+			}
+			else if (args.DataView.Contains(StandardDataFormats.WebLink))
+			{
+				title += " (WebLink)";
+
+				var webLink = await args.DataView.GetWebLinkAsync();
+				details = webLink.ToString();
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Text))
+			{
+				title += " (Text)";
+
+				var text = await args.DataView.GetTextAsync();
+				details = text;
+			}
+
+			// Determine the drop position
+			var pos = args.GetPosition(this.DropBorder);
+			position = "(" + pos.X.ToString("0.00") + ", " + pos.Y.ToString("0.00") + ")";
+
+			this.DropTitleTextBlock.Text = title ?? string.Empty;
+			this.DropPositionTextBlock.Text = position ?? string.Empty;
+			this.DropDetailsTextBlock.Text = details ?? string.Empty;
+
+			return;
+		}
+
+		/// <summary>
+		/// Event handler for when a drag operation enters the <see cref="DropBorder"/>.
+		/// </summary>
+		private void DropBorder_DragEnter(object sender, global::Windows.UI.Xaml.DragEventArgs e)
+		{
+			// We accept only one operation
+			e.AcceptedOperation = DataPackageOperation.Copy;
+
+			this.ShowDragDropDetails(nameof(global::Windows.UI.Xaml.UIElement.DragEnter), e);
+			return;
+		}
+
+		/// <summary>
+		/// Event handler for when a drag operation leaves the <see cref="DropBorder"/>.
+		/// </summary>
+		private void DropBorder_DragLeave(object sender, global::Windows.UI.Xaml.DragEventArgs e)
+		{
+			this.ShowDragDropDetails(nameof(global::Windows.UI.Xaml.UIElement.DragLeave), e);
+			return;
```

Comment:
Codacy found an issue: [Remove this redundant jump.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6334034)